domain/src/main/java/edu/nextstep/camp/calculator/Operation.kt

```diff
@@ -1,14 +1,9 @@
 package edu.nextstep.camp.calculator
 
-data class Operation private constructor(
+data class Operation constructor(
```

Comment:
```suggestion
data class Operation(
```
constructor를 생략해도 될 것 같아요

Reply:
실수군요. 피드백 감사합니다.
8992c8b 커밋에서 반여하였습니다.